zcash_client_backend/src/data_api/wallet.rs

```diff
@@ -984,7 +1002,6 @@ where
                     Some(memo),
                 ))
             }
-            #[cfg(zcash_unstable = "orchard")]
```

Comment:
Why is this removed?

Reply:
`ShieldedProtocol::Orchard` is stabilized, so this is then required for the match to be total.